### ISSUE TITLE
Autographa draft import error due to symbols ">>>" and "<<<"

### ISSUE THREAD

**Issue by bijob89** [friendsofagape/mt2414#274]
# User Story
As a user the translation draft generated by AgMt has symbols ">>>" and "<<<" to denote untranslated token. This causes the text to be unreadable when imported to Autographa. 

# Acceptance Criteria

# Definition of Done

**Comment by joelthe1** [comment]
Yeah, this is a good one. Let's not introduce any extra characters from our side.
